crates/cover_crypt/src/wasm_bindgen/hybrid_cc_aes.rs

```diff
@@ -14,6 +14,7 @@ use cosmian_crypto_core::{
 use js_sys::Uint8Array;
 use wasm_bindgen::prelude::*;
 
+#[allow(dead_code)]
 pub const MAX_CLEAR_TEXT_SIZE: usize = 1 << 30;
```

Comment:
This constant should be taken from `crypto_core` if you need it.

Reply:
removed